model.go

```diff
@@ -1666,11 +1711,17 @@ func modelV11Fields() (schema.Fields, schema.Defaults) {
 	return fields, defaults
 }
 
+func modelV12Fields() (schema.Fields, schema.Defaults) {
+	fields, defaults := modelV11Fields()
+	fields["virtual-host-keys"] = schema.StringMap(schema.Any())
```

Comment:
I don't understand the schema bit here 100% but isn't it a string to string and not schema.Any()?

Reply:
At the top level, model has a virtual-host-keys entry that is an object of type `virtualHostKeys` (see below), so it's not a string.
```
type virtualHostKeys struct {
	Version         int               `yaml:"version"`
	VirtualHostKeys []*virtualHostKey `yaml:"virtual-host-keys"`
}
```